server/service.go

```diff
@@ -268,31 +269,36 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 		parentHashHex = vars["parent_hash"]
 		pubkey        = vars["pubkey"]
 		ua            = UserAgent(req.Header.Get("User-Agent"))
+		accept        = req.Header.Get("Accept")
```

Comment:
perhaps a tad more descriptive name would be useful

```suggestion
		acceptContentTypes        = req.Header.Get("Accept")
```